src/demo/demo/common/conts/consts.ts

```diff
@@ -31,8 +31,8 @@ export const DEF_CONF: IDatePickerConfig = {
   showSeconds: false,
   showTwentyFourHours: false,
   timeSeparator: ':',
-  multipleYearsNavigateBy: 10,
-  showMultipleYearsNavigation: false,
+  secondaryNavigationStep: 10,
```

Comment:
this is a bit problematic to have the same configuration for all calendars (day/month and soon year). Users might want it to be enabled on day picker but not on month picker. We better to have it separated as we can switch between picker types within a single UI. The same is applicable both for secondaryNavigationStep & showSecondaryNavigation. 

Let's change this please. I will do a full review after that change.

Reply:
So making 4 parameters: `secondaryNavigationStepDayView`, `showSecondaryNavigationDayView`, `secondaryNavigationStepMonthView`, `showSecondaryNavigationMonthView` ?

Reply:
yes, I don't see any other way to do it.

Reply:
Done